nav2_system_tests/src/costmap_filters/tester_node.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         elif self.test_type == TestType.SPEED:
             self.speed_it = 0
             # Expected chain of speed limits
-            self.limits = [50.0, 0.0]
+            self.limits = [49.0, 0.0]
```

Comment:
Why lower by 1?

Reply:
There was an issue in *warehouse_speed_mask.png* file. I fixed it.